frontend/packages/data-portal/e2e/metadataDrawer/metadata-drawer-page.ts

```diff
@@ -0,0 +1,389 @@
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
+import { expect } from '@playwright/test'
+import { E2E_CONFIG, translations } from 'e2e/constants'
+import { PageObject } from 'e2e/page-objects/page-object'
+import { isArray } from 'lodash-es'
+import { DeepPartial } from 'utility-types'
+import { _DeepPartialArray } from 'utility-types/dist/mapped-types'
+
+import {
+  Annotations,
+  Datasets,
+  Tiltseries,
+  Tomograms,
+} from 'app/__generated__/graphql'
+import { TestIds } from 'app/constants/testIds'
+import { getDatasetById } from 'app/graphql/getDatasetById.server'
+import { getRunById } from 'app/graphql/getRunById.server'
+import { isFiducial } from 'app/utils/tomograms'
+
+import { DrawerTestData, DrawerTestMetadata } from './types'
+
+export class MetadataDrawerPage extends PageObject {
+  // TODO: (ehoops) Which region does this belong in?
+  public getBoolString(value?: boolean): string {
+    return value ? 'True' : 'False'
+  }
+
+  // #region Navigate
+  // #endregion Navigate
+
+  // #region API
+  public getDatasetTestMetadata({
```

Comment:
Do we think these functions from utils.ts should stay in a separate file, since they're not technically tied to the "page" but rather just generic GQL queries/helpers?